GoogleForCreators/web-stories-wp
# Redesign: Publishing panel

## Context



## Summary



## Relevant Technical Choices
- Uses the DropDownSelect from the design-system for Publish
- Uses `DropDownSelect` in the `AdvancedDropDown` to adjust the style for redesign. This will likely be changed once we have the "real" dropdown for fonts etc.


## To-do

- [ ] Test adjustments.



## User-facing changes
| Before | After |
|------|------|
| <img width="317" alt="Screenshot 2021-03-15 at 16 26 47" src="https://user-images.githubusercontent.com/3294597/111169003-e221e380-85a2-11eb-85ff-50c7c7510073.png"> | <img width="303" alt="Screenshot 2021-03-15 at 16 25 40" src="https://user-images.githubusercontent.com/3294597/111168836-b9015300-85a2-11eb-8429-07bb6e82780a.png"> |


## Testing Instructions
1. Open the Document Panel
2. Verify the Publish panel looks as in [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3483%3A1021)
3. Verify choosing time / author works as expected.
4. Now add a text element and verify that Selecting Font still works as expected.
5. 


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6514
